### PR TITLE
Change default socks proxy port for product tests

### PR DESCRIPTION
**Pull request by electrum**
Use 1180 instead of 1080 to avoid conflicts with the standard port.

### CODE REVIEW

File: presto-product-tests/README.md [hunk 1]
```diff
@@ -410,11 +410,22 @@ been downloaded:
...
-When you change the default socks proxy port (1080), you have to modify the property `hive.metastore.thrift.client.socks-proxy=hadoop-master:1080` in `conf/presto/etc/catalog/hive.properties`.
+You most likely have some application listening on port `1180` on either
+docker-machine or on your local machine if you are running docker natively.
+You can override the default socks proxy port (`1180`) used by dockerized
+Hive deployment in product tests using the `HIVE_PROXY_PORT` environment
+variable, e.g. `export HIVE_PROXY_PORT=5555`. This will run all of the
+dockerized tests using the custom port for the socks proxy.
+
+When you change the default socks proxy port (`1180`), you have to modify
+the property `hive.metastore.thrift.client.socks-proxy=hadoop-master:1180`
```
Comment (maciejgrzybek) [inline presto-product-tests/README.md]: Actually I made it a bit unclear here.
Product tests running from inside docker will have to still communicate on 1080 but from OUTSIDE of docker container (e.g. if you run product tests from your IntelliJ or connect from outside docker Presto to Hive instance [not for product tests]), then the property has to change.

To make it less confusing I'm about to start a PR changing default port inside docker soon.
Then we will have mapping 1180:1180 by default (socks proxy accessible the same way inside and outside docker) and if non-standard configuration is going to be used, then we will end up having 1180:whatever mapping.

Similar change was introduced for MySQL and Postgres.

File: presto-product-tests/conf/presto/etc/catalog/hive.properties [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 connector.name=hive-cdh5
 hive.metastore.uri=thrift://hadoop-master:9083
-hive.metastore.thrift.client.socks-proxy=hadoop-master:1080
+hive.metastore.thrift.client.socks-proxy=hadoop-master:1180
```
Comment (maciejgrzybek) [inline presto-product-tests/conf/presto/etc/catalog/hive.properties]: Please mind my previous comment.
This change will be correct once I publish new docker image.

Please remove that line and change comment accordingly.
I will put the configuration back to `1180` when docker image with socks proxy listening on `1180` is published.

Comment (electrum) [inline presto-product-tests/conf/presto/etc/catalog/hive.properties]: Sounds good. I'll wait for you to publish the new image. If you want to take over the PR, feel free.

Comment (maciejgrzybek) [inline presto-product-tests/conf/presto/etc/catalog/hive.properties]: I've just built and published new images. I've just run tests against new docker image. Will start a PR once they pass so you can piggy back on that in your PR or take mine instead.

Comment (maciejgrzybek) [inline presto-product-tests/conf/presto/etc/catalog/hive.properties]: Here's my PR: https://github.com/prestodb/presto/pull/6283.
Feel free to amend your changes or merge the other one.

**Pull request closed without merging**